blocks/core/ff_container/ff_container-expandable/ff_container-expandable.xsl

```diff
@@ -0,0 +1,20 @@
+<xsl:template name="ff_container-expandable">
+	<xsl:param name="data" />
+	<div class="ff_container-expandable">
+		<div class="ff_container-expandable__dropdown ff_container-expandable__dropdown--small-screens" data-ff-action="expandable">
```

Comment:
Do you need the `class=""` in here if you're overwriting it with `xsl:attribute` ?